helm-charts/FATE/values-template-example.yaml

```diff
@@ -26,6 +26,12 @@ modules:
 
 backend: eggroll
 
```

Comment:
I think we could add some comments here, to make user know what do these commented line do.

Reply:
We can add features about adding comments